swift/ql/src/queries/Security/CWE-259/ConstantPassword.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>Deriving password-based encryption keys using hard-coded passwords is insecure, because the generated key may be easily discovered. Data hashed using constant salts are vulnerable to dictionary attacks, enabling attackers to recover the original input.</p>
```

Comment:
I think we use `hardcoded` (one word) in our query help, but I could be wrong 🤔 
Also you use one word only on line 7, so probably best to be consistent.
```suggestion
    <p>Deriving password-based encryption keys using hardcoded passwords is insecure, because the generated key may be easily discovered. Data hashed using constant salts are vulnerable to dictionary attacks, enabling attackers to recover the original input.</p>
```

Reply:
Agreed. Fixed as suggested.